### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54296]
#### VFS team name
Sitewide Facilities

#### VFS product name
VAMC

#### VFS feature name
VAMC Lovell Federal Health care

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
There are several instances where the address is buried in content and does not have the address block applied to it.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/52672_allison_040703.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Add the address block when addresses are buried in content. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by davidmpickett** [comment]
The inconsistent formatting of addresses seems to be a problem across VAMCs and would need to be addressed be Content Type hardening. For instance: The TRICARE version of the Medical Records page seems to be formatting addresses correctly. Compare with the Lovell VA page and a non-Lovell VAMC.

https://www.va.gov/lovell-federal-health-care-tricare/medical-records-office/
https://www.va.gov/lovell-federal-health-care-va/medical-records-office/
https://www.va.gov/chicago-health-care/medical-records-office/
